.hooks/bootstrap

```diff
@@ -90,7 +90,7 @@ if [ -n "${CI}" ]; then
     echo "Detected CI environment. Running extra requirements"
     echo "Check if poetry files are valid"
     poetry check --no-interaction
-    exit_on_error $? "Failed to check poetry files, exiting with exit code $?"
+    exit_on_error $? "Failed to check poetry files"
```

Comment:
all the fixes in this file are because `exit_on_error` already prints the error code, so no need to do it here.